guides/overview.md

```diff
@@ -4,9 +4,135 @@ Amber is designed to make common web-development tasks simple, and easy, maintai
+
+* The [Crystal ](https://crystal-lang.org/)language version 0.23.1 or higher
+* The [Shards](https://github.com/crystal-lang/shards) packaging system, which is installed with Crystal by default.
+* A working installation of the [SQLite3 Database](https://www.sqlite.org/)
+
+Amber is a web application framework running on the Crystal programming language. If you have no prior experience with Crystal, you will find a very steep learning curve diving straight into Amber. There are several curated lists of online resources for learning Crystal:
+
+* [Official Learn Crystal Docs](https://crystal-lang.org/docs/overview/)
+
+Be aware that the Crystal language is under development
+
+## What is Amber?
+
+Amber is a web application development framework written in the Crystal language. It is designed to make programming web applications easier by making assumptions about what every developer needs to get started. It allows you to write less code while accomplishing more than many other languages and frameworks. Experienced Amber developers also report that it makes web application development more fun.
+
+Amber is opinionated software. It makes the assumption that there is a "best" way to do things, and it's designed to encourage that way - and in some cases to discourage alternatives.
+
+The Amber philosophy includes two major guiding principles:
+
+* **Don't Repeat Yourself:**
+  DRY is a principle of software development which states that "Every piece of knowledge must have a single, unambiguous, authoritative representation within a system." By not writing the same information over and over again, our code is more maintainable, more extensible, and less buggy.
+* **Convention Over Configuration:**
+  Amber provides a basic organizational structure that covers class names, filenames, database table names, and other conventions. While the conventions take some time to learn, by following the conventions Amber provides you can avoid needless configuration and make a uniform application structure that makes working with various projects simple.
+
+## Creating a New Amber Project
+
+Open up a command line prompt. On macOS open Terminal.app, on Windows choose "Run" from your Start menu and type `cmd.exe`. Any commands prefaced with a dollar sign `$` should be run in the command line. Verify that you have a current version of Ruby installed:
+
+```bash
+$ crystal -v
+Crystal 0.23.1 (2017-09-08) LLVM 4.0.1
+```
+
+Many popular UNIX-like OSes ship with an acceptable version of SQLite3. On Windows, if you installed Rails through Rails Installer, you already have SQLite installed. Others can find installation instructions at the [SQLite3 website](https://www.sqlite.org/). Verify that it is correctly installed and in your PATH
+
+```bash
+$ sqlite3 --version
+```
+
+The program should report its version.
+
+### Installing Amber
+
+To install the Amber binary on Mac you can use
+
+```bash
+$ brew install amberframework/amber/amber
+```
```

Comment:
This should now be:
```
brew install amberframework/amber/amber
```

@elorest - I don't know anything about creating homebrew packages. Is there any reason not to switch to:

```
brew install amberframework/amber
```

Based on @marksiemers comment: https://github.com/amberframework/docs/pull/43/files#r154249657